index.html

```diff
@@ -1,46 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
 <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="styles/reset.css">
     <link rel="stylesheet" href="styles/style.css">
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
-</head>
-
-<html>
-
-<body>
-    <header>
-        <nav>
-            <div class="topnav">
-                <a class="active" href="#about">About</a>
-                <a href="#store">Store</a>
-                <div class="topnav-right">
-                    <a href="#gmail">Gmail</a>
-                    <a href="#images">Images</a>
-                    <a href="#"><span class="material-icons">
-                            apps
-                        </span></a>
-                    <a href="https://github.com/micheaol" target="_blank"> <span class="material-icons">
-                            account_circle
-                            </span>
-                    </a>
+    <title>Review trial</title>
+</head>    
+    <body>
+        <header>
+            <nav>
+                <div class="topnav">
+                    <a class="active" href="#about" rel="noopener">About</a>
+                    <a href="#store">Store</a>
+                    <div class="topnav-right">
+                        <a href="#gmail">Gmail</a>
+                        <a href="#images">Images</a>
+                        <a href="#" rel="noreferrer"><span class="material-icons">
+                                apps
+                            </span></a>
+                        <a href="https://github.com/micheaol" target="_blank"  rel="noopener"> <span class="material-icons">
+                                account_circle
+                                </span>
+                        </a>
+                    </div>
                 </div>
-            </div>
-        </nav>
-    </header>
-    <main>
-        <form role="search">
-            <img src="images/googlelogo_color_272x92dp.png">
-            <div>
-                <input type="search">
-                <button id="mic"> <span class="material-icons">
-                    keyboard_voice
-                    </span> </button>
-            </div>
-            <div>
-                <button> Google Search </button>
-                <button> I'm Feeling Lucky </button>
-            </div>
-        </form>
-    </main>
+            </nav>
+        </header>
+        <main>
+    
+            <form role="search">
+                <img src="images/googlelogo_color_272x92dp.png" alt="image">
+                <div>
```

Comment:
Please add rel to your image tag in line 31 because that will help with the screen reader.